site/_resources/release-process.md

```diff
@@ -20,7 +20,7 @@ The steps for an alpha or beta release are
 - Tag the head of master with the relevant release tag (in this case `alpha.1`), and push
 
 ```sh
-$ git tag -a v0.15.0-alpha.1 -m 'contour 0.15.0 alpha 1'
+$ git tag -a v0.15.0-alpha.1 -m 'contour 1.2.0 alpha 1'
```

Comment:
Maybe some of these can be replaced with `{{site.latest}}`?

Reply:
Sure, seems like a good idea.